web/packages/README.md

```diff
@@ -8,6 +8,7 @@ a stand-alone web application or library referenced by other packages.
 | Package    | Description                                                                  |
 | ---------- | ---------------------------------------------------------------------------- |
 | `teleport` | Open-source version of Gravitational Teleport Web UI                         |
+| `teleterm` | The Electron app of Teleport Connect                                         |
```

Comment:
Maybe we can align it with Web UI (as there is a closed-source version too)?
```suggestion
| `teleterm` | Open-source version of Teleport Connect (Electron app)                                       |
```

IMO I'd also drop `Gravitational` in the package descriptions.

Reply:
> Maybe we can align it with Web UI (as there is a closed-source version too)?

Good idea, I think I wrote what I wrote because technically we have a single version of Connect at the moment.